### PR TITLE
WIP: litestream sqlite backup service

### PR DESCRIPTION
**Pull request by hlolli**
Some experimentations on the docker-compose config and configuration setup for easy configuration integration.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
----
-version: '3.0'
+version: '3.8'
+
+volumes:
+  chunks-data:
+  contiguous-data:
+  headers-data:
+  db-data:
```
Comment (djwhitt) [inline docker-compose.yaml:7]: I like this structure, but we need to continue to use volume mounts specified by environment variables by default for these.

File: litestream.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+dbs:
```
Comment (djwhitt) [inline litestream.yaml:1]: Checkout how our Envoy container renders a template on startup: https://github.com/ar-io/ar-io-node/blob/develop/envoy/docker-entrypoint.sh#L12

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,5 +1,10 @@
----
-version: '3.0'
+version: '3.8'
```
[the remark this answers is not in the document]
Comment (hlolli) [reply docker-compose.yaml:1]: I'm not sure, I was following the documentation of 3.8~3.9 and not 3.0. I'm happy to revert this.

File: litestream.yaml (hunk 2 above)
Comment (hlolli) [reply litestream.yaml:1]: it's been adjusted to follow the envoy pattern

File: docker-compose.yaml (hunk 1 above)
Comment (hlolli) [reply docker-compose.yaml:7]: I need to look deeper into combining those two 👍

File: litestream.yaml (hunk 2 above)
Comment (hlolli) [reply litestream.yaml:1]: no what am I saying :D I need to first push my local changes 🤦

**Comment by hlolli** [comment]
I did git mistakes and I continue this here: https://github.com/ar-io/ar-io-node/pull/113

**Pull request closed without merging**